Class-17-(TypeScript).md

```diff
@@ -0,0 +1,36 @@
+## In the prep work this week (TypeScript), the students learned:
+
+1. What TypeScript is and how it is different from JavaScript.
+1. The differences between static and dynamic typing.
+1. How to declare and use:
+    1. Variables of number, string, and boolean types.
+    1. Arrays and tuples.
+    1. Classes and interfaces.
+1. How to use export and import.
+1. How to use the ``implements`` keyword to make classes implement interfaces.
+
+## Large Group Time (Instructor Notes)
+
+### Announcements
+
+1. Graded assignment #6 opens soon. This is the last graded assignment for unit 1.
+1. Remind the students about the requirements for remaining in the course after unit 1.
+1. Check with your class Candidate Engagement Manager for any other announcements.
+
+### Topics That Require More Attention
+
+1. Be ready to answer the question, "Why do we need to learn TypeScript?"
+1. Q & A as needed for the prep-work exercises.
+1. Review the distinction between *classes* and *interfaces* in TypeScript.
+1. Review the syntax for function declarations in TypeScript:
+    1. The type must be identified for each parameter.
+    1. If a value is returned, the type for that value must also be identified.
+1. Note that when a TypeScript file is compiled, the process generates a JavaScript file that can be run.
+1. Studio intro.
+
+## Studio (TF Notes)
+
+1. Remind the students that they need to be in the master branch of the TypeScript repository. Also remind them to switch to the "studio" folder to find the starter code.
```

Comment:
I would add some clarity to this. If I had not read the actual studio instructions, I would not know that this was a commentary on how to properly fork the repo.

Reply:
Hmmm.  This was not actually a commentary about forking the repo, which just reinforces the need for clarity.

I split this into two revised bullet points.  Let me know what you think.